src/ConsoleApp/Commands/Modules/Debug.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Threading.Tasks;
 using Discord.Commands;
+using StupifyConsoleApp.Client;
```

Comment:
New using without any other changes to file

Reply:
ye was testing something, deleted the function after the test, kept the using coz idk I may use it later again lol